### ISSUE TITLE
initial_split_orientation not work after Rev. ​5235145​

### ISSUE THREAD

**Issue by ZoomQuiet** [leo-editor/leo-editor#267]
from [Why outline auto slide left? · Issue #265 · leo-editor/leo-editor](https://github.com/leo-editor/leo-editor/issues/265)

- when i upgrade local leo into github last version
- `myLeoSettings.leo#@settings-->Windows-->Splitter options-->@string initial_split_orientation = horizontal`
- not working, the body pane alwasy as bottom

![screenshot 2016-04-20 11 35 56](https://cloud.githubusercontent.com/assets/22494/14682622/186f3e96-06ec-11e6-97e2-74159fd24c43.png)

**Comment by wangzhaohe** [comment]
Leo 5.3-devel, build 20160412153848, Tue Apr 12 15:38:48 CDT 2016
The same problem.

**Comment by edreamleo** [comment]
I'm working on this now. The problem is likely in dw.setSplitDirection.  It now uses general code to get splitters, but that doesn't work early in the startup process. I expect to fix this within a day.

**Comment by edreamleo** [comment]
Fixed at b0b473d. Please reopen if there are further problems.

**Comment by SegundoBob** [comment]
Thanks.

I verified the fix using revision e89401da4f57.

# Test System
```
Xubuntu 64-bit 14.04
Leo 5.3-devel, build 20160412153848, Tue Apr 12 15:38:48 CDT 2016
Git repo info: branch = master, commit = e89401da4f57
Python 3.4.3, PyQt version 5.2.1
```

**Comment by wangzhaohe** [comment]
It is OK now! Thanks!
